presto-main/src/main/java/com/facebook/presto/operator/GroupByHash.java

```diff
@@ -85,4 +88,34 @@ default boolean contains(int position, Page page, int[] hashChannels, long rawHa
 
     @VisibleForTesting
     int getCapacity();
+
+    static IntIterator getHashSortedGroupIds(GroupByHash groupByHash)
```

Comment:
make it non-static
```java
    default IntIterator getHashSortedGroupIds()
    {
        IntBigArray groupIds = new IntBigArray();
        groupIds.ensureCapacity(getGroupCount());
        for (int i = 0; i < getGroupCount(); i++) {
            groupIds.set(i, i);
        }

        groupIds.sort(0, getGroupCount(), (leftGroupId, rightGroupId) -> Long.compare(getRawHash(leftGroupId), getRawHash(rightGroupId)));

        return new AbstractIntIterator()
        {
            private final int totalPositions = getGroupCount();
            private int position;

            @Override
            public boolean hasNext()
            {
                return position < totalPositions;
            }

            @Override
            public int nextInt()
            {
                return groupIds.get(position++);
            }
        };
    }

Reply:
I see that `default` methods allow the implementing class to override, whereas `static` methods are  NOT overridable. Will update to `default`